tox.ini

```diff
@@ -7,8 +7,8 @@ envlist =
     py27-1.11,
     py34-{1.11,2.0},
     py35-{1.11,2.0,2.1,2.2},
-    py36-{1.11,2.0,2.1,2.2,master},
-    py37-{2.0,2.1,2.2,master}
+    py36-{1.11,2.0,2.1,2.2,3.0,master},
+    py37-{2.0,2.1,2.2,3.0,master}
 
```

Comment:
Django 3.0 officially support python 3.8. It should probably be added to the test matrix.